src/main/java/com/dynatrace/loadrunner/CommandLineParser.java

```diff
@@ -0,0 +1,56 @@
+package com.dynatrace.loadrunner;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class CommandLineParser {
+
+	final public Map<String, String> arguments;
+
+	public CommandLineParser(String[] args) {
+
+		arguments = new HashMap<>();
+
+		for (int i = 0; i < args.length; i++) {
```

Comment:
`args[]` processing should be extracted to a separate method, so the usage looks like:
```
CommandLineParser parser = new CommandLineParser();
parser.parseArguments(args);
...
```
this way you can throw exception in case when the parsing finds a problem, and it's testable
do we even need this class to be stateful? Couldn't we use some utility class, like:
```
Map<String, String> parsedArgs = CommandLineParser.parseArguments(args);
```